recipes/diffpy-structure/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "diffpy.structure" %}
+{% set version = "1.3.4" %}
+{% set sha256 = "de481329c4dc41ce1538f88fac768c8f677860602caa886fbfb3429bc17ac3ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pycifrw
+  run:
+    - python
+    - setuptools
+    - numpy >=1.3
+    - pycifrw
+
+test:
+  imports:
+    - diffpy
+    - diffpy.Structure
+    - diffpy.Structure.Parsers
+    - diffpy.Structure.applications
+    - diffpy.Structure.expansion
+    - diffpy.Structure.tests
+
+about:
+  home: https://github.com/diffpy/diffpy.structure
+  license: BSD
```

Comment:
@CJ-Wright Can probably call this `license: BNL` and leave the `license_family: BSD`